politeiawww/invoices.go

```diff
@@ -1592,6 +1594,20 @@ type backendInvoiceStatusChange struct {
 	Timestamp      int64              `json:"timestamp"`      // Timestamp of the change
 }
 
+// backendInvoicePayment represents an invoice payment and is stored
+// in the metadata stream mdStreamInvoicePayment in politeiad.
+type backendInvoicePayment struct {
+	Version        uint               `json:"version"`        // Version of the struct
+	Token          string             `json:"token"`          // Token of the associated invoice
+	Address        string             `json:"address"`        // Payment address of the invoice
+	TxIDs          string             `json:"txids"`          // TxIDs captured from the payment
```

Comment:
Comment should denote how the string is formatted.  Is it comma seperated txids?